config/install/core.entity_view_display.node.stanford_person.default.yml

```diff
@@ -401,7 +402,7 @@ third_party_settings:
             region: right
             configuration:
               id: 'field_block:node:stanford_person:su_person_telephone'
-              label: Telephone
+              label: Contact
```

Comment:
@cjwest The contact label is in the config file. I'm not sure why it isn't merged on the pages you mention.